test/Tryouts/Program.cs

```diff
@@ -1,377 +1,25 @@
-using Raven.Client.Documents;
-using RachisTests.DatabaseCluster;
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
+/*
```

Comment:
Move this to a separate file, there is too much churn here.

Reply:
Good idea, done